report/src/presence/bluetooth.tex

```diff
@@ -32,11 +32,14 @@ \subsubsection{Bluetooth Low Energy} \label{sec:bluetooth_low_energy}
 This is because most use cases require LE Coded PHY, a feature of BLE that improves the range and robustness of the wireless communication at the cost of a lower data transfer rate.\cite{BluetoothOverview}\cite{BLE_Regulatory_Aspects_Document}
 
 While BLE was known for its device communication capabilities, it is now widely used as a device positioning technology due to the increasing demand for accurate indoor location services.
-Since its release, BLE has evolved to include features enabling it to detect the presence, distance, and direction of nearby devices by leveraging the Received Signal Strength Indication (RSSI) of BLE.
+Similar to Wi-fi, BLE can be used to detect the presence, distance, and direction of nearby devices by leveraging the Received Signal Strength Indication (RSSI) of the BLE device.
 SIG themselves recommend using BLE for presence detection in indoor environments because of its low power consumption and high accuracy.\cite{BluetoothOverview}\cite{BLE_Regulatory_Aspects_Document}
```

Comment:
```suggestion
SIG themselves recommend using BLE for presence detection in indoor environments because of its low power consumption and high accuracy.\cite{BluetoothOverview}\cite{BLE_Regulatory_Aspects_Document}
```
Should the cite note be of the form \cite{BluetoothOverview, BLE_Regulatory_Aspects_Document}?